engine/src/main/scala/cromwell/engine/workflow/WorkflowMetadataBuilder.scala

```diff
@@ -2,24 +2,24 @@ package cromwell.engine.workflow
 
 import akka.actor.ActorSystem
 import cromwell.core.WorkflowId
-import cromwell.database.obj.{WorkflowExecution, WorkflowExecutionAux}
+import cromwell.database.obj.WorkflowExecution
 import cromwell.engine
 import cromwell.engine._
-import cromwell.engine.backend.CallMetadata
-import cromwell.engine.db.DataAccess.WorkflowExecutionAndAux
+import cromwell.engine.backend.{CallMetadata, WorkflowDescriptor}
 import cromwell.engine.db._
 import cromwell.engine.finalcall.FinalCall._
-import cromwell.engine.workflow.WorkflowManagerActor._
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowNotFoundException
 import cromwell.webservice._
 import org.joda.time.DateTime
 import spray.json._
 import wdl4s._
 
-import scala.concurrent.{ExecutionContext, Future}
-import scala.io.Source
+import scala.concurrent.Future
 
 object WorkflowMetadataBuilder {
 
+  type DBMap[+V] = Map[ExecutionDatabaseKey, V]
```

Comment:
I prefer to avoid type aliases like this in favor of case classes but I'm also used to people telling me no on that one :)